test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -350,6 +415,82 @@ test.skip
     await t.expect(identityPage.getIdentifierValue()).eql('Test Identifier');
   });
 
+test
+  .requestHooks(loopbackSuccessLogger, loopbackSuccessWithHttpsMock)('in loopback server approach, https loopback succeeds', async t => {
+    failureCount = 0;
+    // after OKTA-715718 is fixed, should use ".eql(0)" for ".lte(otherProbeCount)" assertions
+    const otherProbeCount = process.env.OKTA_SIW_GEN3 === 'true' ? 0 : 1;
```

Comment:
bacon test [flakiness](https://bacon-go.aue1e.saasure.net/commits?artifact=okta-signin-widget&branch=OKTA-719439-https-loopback&page=1&pageSize=6&sha=30742a3e24fa0e04fc2764c5edcc2593764245fc&tab=tests&testSuite=okta-signin-widget-testcafe)

@denysoblohin-okta has a fix in this [PR](https://github.com/okta/okta-signin-widget/pull/3654)

According to my local test, it is indeed flaky, especially when I add break points

SIW V2, should be only two `/probe` call like V3
but actually, four `/probe` calls
![Screenshot 2024-06-14 at 11 54 49 AM](https://github.com/okta/okta-signin-widget/assets/171885610/e520202a-bb12-4d91-a03a-6992908287bf)

three `/probe` calls
![Screenshot 2024-06-14 at 11 50 10 AM](https://github.com/okta/okta-signin-widget/assets/171885610/b4e4e691-283b-4418-bb8a-457dc439f410)

SIW V3, it is using simple async/await, so only two `/probe` calls, which is expected
![Screenshot 2024-06-13 at 1 55 53 PM](https://github.com/okta/okta-signin-widget/assets/171885610/37bd13eb-dd9c-49b2-b27e-202bd4901008)

Reply:
any specific reasons we see different number of probe calls in V2 and V3? I'm fine with asserting different number of calls but would like to understand the root cause

Reply:
I believe the reason is in this [PR](https://github.com/okta/okta-signin-widget/pull/3654)'s description. Mainly because Promisified $.ajax in BaseOktaVerifyChallengeView were not chained correctly